### DISCUSSION TITLE
react-navigation-material-top-tabs-inside-scrollview-gives-virtualizedlists-should-never-be-nested-inside-plain-scrollviews-error

### DISCUSSION THREAD

**Discussion by rosnk** [react-navigation/react-navigation#10695] [Q&A]
Intention: I want a scrollview with contents and between the content i also want top tab navigation view. It works but it also gives following error.

using “createMaterialTopTabNavigator” from “@react-navigation/material-top-tabs” gives following error:
“VirtualizedLists should never be nested inside plain ScrollViews with the same orientation because it can break windowing and other functionality - use another VirtualizedList-backed container instead”

relevant code snippet from my codebase:

```
import {
  createMaterialTopTabNavigator,
} from "@react-navigation/material-top-tabs";

const Tab = createMaterialTopTabNavigator();

const DetailScreen = ({ route }) => {
  return (
      <SafeAreaView>
        <ScrollView style={styles.scrollView} nestedScrollEnabled={true}>
        <View><Text>different content block</Text></View>
        <View>
            <Tab.Navigator>
              <Tab.Screen name="tab1" component={tab1} />
              <Tab.Screen name="tab2" component={tab2} />
            </Tab.Navigator>
          </View>
       </ScrollView>
     </SafeAreaView>

});
```

SDK Version: “expo”: “^44.0.0”,
Platforms(Android/iOS/web/all): all

**Comment by ahaseeb001** [comment]
same issu ,tried many methods but no solution

**Reply by raajnadar** [reply]
https://github.com/react-navigation/react-navigation/discussions/10695#discussioncomment-3531481

**Comment by raajnadar** [comment]
Why not have ScrollViews inside each tab? This way you can prevent the warning

**Reply by muyi0327** [reply]
> Why not have ScrollViews inside each tab? This way you can prevent the warning

I have a long page, and the tab is at the bottom of the page. I must scroll to see the tab.

**Reply by raajnadar** [reply]
So add the scroll view inside the tab screen, read my comment again

**Reply by muyi0327** [reply]
> So add the scroll view inside the tab screen, read my comment again

First，Tabs are not in the view. You must scroll to see them
